driver/src/main/java/org/cf/oracle/Driver.java

```diff
@@ -92,13 +91,33 @@ private static void showUsage() {
     }
 
     public static void main(String[] args) {
+        // find out the driver dir as it is different on some devices
+        for (String driverDir : new String[] { "/data/local/tmp", "/data/local" }) {
```

Comment:
Why not just assume files are in the same directory? Or get current working directory? It's cleaner than hard coding temp paths in two places.